libs/shared/auth/src/lib/user-info-container/user-info-container.component.spec.ts

```diff
@@ -4,7 +4,8 @@ import { render, RenderResult, screen } from '@testing-library/angular';
 import { KeycloakService } from 'keycloak-angular';
 import { KeycloakProfile } from 'keycloak-js';
 import { provideMockStore } from '@ngrx/store/testing';
-import { AuthSelectors } from '@expense-tracker-ui/shared/auth/data-access';
+import { AuthSelectors } from '../../index';
+import '@testing-library/jest-dom';
```

Comment:
why is this needed?

Reply:
For some reason expect(...).toBeInTheDocument() was not being recognized. Adding the import statement fixed the issue.
It's strange though...